Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM quay.io/ansible/python-builder:latest as builder
 # =============================================================================
 
+ARG ANSIBLE_BRANCH=""
 ARG ZUUL_SIBLINGS=""
 COPY . /tmp/src
+RUN if [ "$ANSIBLE_BRANCH" != "" ] ; then \
+      echo "Installing requirements.txt / upper-constraints.txt for Ansible $ANSIBLE_BRANCH" ; \
+      cp /tmp/src/tools/requirements-$ANSIBLE_BRANCH.txt /tmp/src/requirements.txt ; \
+      cp /tmp/src/tools/upper-constraints-$ANSIBLE_BRANCH.txt /tmp/src/upper-constraints.txt ; \
+    fi
```

Comment:
So we need to do this, because of ansible / ansible-base / ansible-core pypi naming.  For stable-2.9, we need to use ansible.  But for stable-2.10, I think we ship ansible-base, then once we release stable-2.11, it is ansible-core.

So confusing!

Reply:
that really is painful.